### PR TITLE
[ffmpeg-qsv] add jpeg encode playback_seek test case

### PR DESCRIPTION
**Pull request by Hangjie22Coder**

### CODE REVIEW

File: test/ffmpeg-qsv/encode/jpeg.py [hunk 1]
```diff
@@ -31,3 +31,21 @@ def test_r2r(self, case, quality):
...
+      bitrate   = bitrate,
+      maxrate   = maxrate,
+      minrate   = bitrate,
+      rcmode    = rcmode,
+      fps       = fps,
+      seek      = seek,
+    )
+
+  @slash.parametrize(*gen_jpeg_seek_parameters(spec))
+  def test(self, case, rcmode, bitrate, maxrate, fps, seek):
```
Comment (uartie) [inline test/ffmpeg-qsv/encode/jpeg.py:49]: The bitrate and maxrate don't make sense for jpeg.
Also, only cqp mode for jpeg.  So rcmode param is not needed, either.
Hence, jpeg will need a special param generator.

**Review by uartie — CHANGES_REQUESTED** [review]

File: test/ffmpeg-qsv/encode/jpeg.py (hunk 1 above)
Comment (Hangjie22Coder) [reply test/ffmpeg-qsv/encode/jpeg.py]: Thank you for your review and suggestions. I have updated the code by removing the `rcmode`, `bitrate`, and `maxrate `elements as per your recommendation. The modified implementation now focuses only on the necessary parameters (`fps`, and `seek`). Please let me know if further adjustments are needed.

**Review by FocusLuo — APPROVED** [review]
LGTM

**Comment by FocusLuo** [comment]
@Hangjie22Coder  could you help to fix the conflict?

**Review by uartie — APPROVED** [review]

**Pull request merged**